### DISCUSSION TITLE
E3V2-UBL-BLTouch-15x15-HS-v4.2.7-v1.3.4

### DISCUSSION THREAD

**Discussion by MarcDuchesne001** [Jyers/Marlin#1008] [General Discussion]
Hi everyone and thanks for all your hard work, we appreciate.

I am in the process to compiling my own .bin file since I have purchased an Ender 3 extender. It is basically an ender 3 that I upgraded to a V2 (4.2.7 board) with proper screen. 

I have replaced the Marlin configuration.h and configuration_adv.h (UBL-BLTouch-10x10) and edited these to fit my needs for 4.2.7 board.

I am not finding info's about the HS function, if it is enables or not on these conf files.

Any help, hoping I am clear in explanations :-) 

Thanks !
-marc

**Comment by MarcDuchesne001** [comment]
Just find this info. Simply enable #define BLTOUCH_HS_MODE in configuration_adv.h for high speed probing. Will take a look tonight Is this ok or more is to be setup ?

**Comment by smooreace** [comment]
I think that's all that's needed, but why not start with a config
sample that better fits what you were after? (BLTouch-3x3-HighSpeed)

**Comment by smooreace** [comment]
Or, you can compare the different templates in Notepad++ and see exactly
what is different between them to better understand how each feature is
activated

**Reply by MarcDuchesne001** [reply]
Cool, will do and keep posted. Thanks !!!

**Reply by MarcDuchesne001** [reply]
I also modded the bed size in the conf file for 400X400 and a few other things... Since this is a larger bed, I would like more points like 10X10 or 15X15 taken.

**Comment by smooreace** [comment]
Along those lines, I just finished pushing my v2 to 400x400x500 with BL
Touch and MicroSwiss DD/Hot End. I THINK I'm still printing a few MM
off-center, but SOOO much room to work now!
